crowbar_framework/app/controllers/application_controller.rb

```diff
@@ -120,15 +121,23 @@ def log_exception(e)
 
   def render_not_found
     respond_to do |format|
-      format.html { render "errors/not_found", status: 404 }
-      format.json { render json: { error: "Not found" }, status: 404 }
+      format.html do
+        render "errors/not_found", status: :not_found
+      end
+      format.json do
+        render json: { error: I18n.t("error.not_found") }, status: :not_found
+      end
     end
   end
 
   def chef_is_offline
     respond_to do |format|
-      format.html { render "errors/chef_offline", status: 500 }
-      format.json { render json: { error: "Chef server is not available" }, status: 500 }
+      format.html do
+        render "errors/chef_offline", status: :internal_server_error
+      end
+      format.json do
+        render json: { error: I18n.t("error.chef_server_not_available") }, status: :internal_server_error
```

Comment:
Metrics/LineLength: Line is too long. [105/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)